PRS4001-5000/PRS4822Gregory.xml

```diff
@@ -45,18 +45,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
          <change who="PL" when="2019-03-27">removed occupation containing St, Saint, bibl, biblical or prophet, added keyword. See https://github.com/BetaMasaheft/Documentation/issues/1104</change>
+  <change when="2020-03-23" who="ES">added names and wikidata id</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
-               <persName>
-                  
-                  Gregory of Nazianzos
-               </persName>
-               <persName type="alt">Gorgoryos</persName>
-               <!-- St -->
+            <person sex="1" sameAs="">
```

Comment:
wikidata :)

Reply:
oops… :) done

Reply:
just pushed another item sorry confused branches (too many… and my home connection is a bit dreamy)

Reply:
Ah, I know the problem :)